README.md

```diff
@@ -8,6 +8,7 @@ Steps to use this template:
 - Click on *Create repository*
 - Update the placeholders (`git grep -i CHANGEME`, `git grep -i TODO`)
```

Comment:
```suggestion
- Update the placeholders (`git grep -i CHANGEME`, `git grep -i TODO`, `git grep -i mytracer`)
```
Either update the grep commands, or rename to `MyTracer` to `ChangeMeTracer` in the go files.